### DISCUSSION TITLE
Installing jaxlib

### DISCUSSION THREAD

**Discussion by herobd** [jax-ml/jax#8307] [Q&A] [answered]
I've tried installing jaxlib with `pip install jaxlib`, but I just get:
```
ERROR: Could not find a version that satisfies the requirement jaxlib (from versions: none)
ERROR: No matching distribution found for jaxlib
```
Where do I get jaxlib?

**Comment by jakevdp** [comment, marked as answer]
Full installation information is available in the README: https://github.com/google/jax#installation

If the instructions there do not work, it may be that you are on an unsupported platform (e.g. Windows) or using an unsupported Python version (e.g. Python 3.6 or 3.10)

**Reply by herobd** [reply]
Thanks. I was on Python 3.10. Where does it say which Python versions are supported? The Pypi page for jaxlib just says "Requires: Python >=3.7"

**Reply by hawkinsp** [reply]
Python 3.10 is actually supported. You just have to build jaxlib from source. We can't easily release a jaxlib wheel ourselves until `scipy` release a Python 3.10 wheel; note you'd also have to build `scipy` from source for Python 3.10.

**Reply by herobd** [reply]
Ah, okay. Just downgrading Python is an easy solution..
Thanks!
